### PR TITLE
Register Subdomain: Nodirbek

### PR DESCRIPTION
**Pull request by Nodirbek298**
Added the domain: Nodirbek

### CODE REVIEW

File: domains/Nodirbek.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "Piratebay298",
+        "email": "piratebay298@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "https://github.com/Piratebay298/"
```
Comment (WilliamDavidHarrison) [inline domains/Nodirbek.json:9]: CNAME cannot contain `https://` or `/`. I would recommend changing this to a `URL` record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/Nodirbek.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "https://github.com/Piratebay298/"
```
Comment (WilliamDavidHarrison) [inline domains/Nodirbek.json]: image
I just get a white screen.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/Nodirbek.json (hunk 2 above)
Comment (Piratebay298) [reply domains/Nodirbek.json]: Well fixed it

**Review by wdhdev — DISMISSED** [review]

File: domains/Nodirbek.json (hunk 2 above)
Comment (Piratebay298) [reply domains/Nodirbek.json]: > image I just get a white screen.

there was missing index.html

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**